livekit-rtc/livekit/rtc/room.py

```diff
@@ -112,16 +113,19 @@ def __init__(self, loop: Optional[asyncio.AbstractEventLoop] = None) -> None:
         self._info = proto_room.RoomInfo()
 
         self.participants: Dict[str, RemoteParticipant] = {}
```

Comment:
new api is using remote_participants I think

Reply:
let's make sure we're aligned with other SDKs, we can only break once if we release v1